com.unity.netcode.gameobjects/Runtime/Connection/NetworkConnectionManager.cs

```diff
@@ -925,6 +935,7 @@ internal void OnClientDisconnectFromServer(ulong clientId)
                         }
                     }
                     else
+                    if (!NetworkManager.ShutdownInProgress)
```

Comment:
Any particular reason you're putting all these ifs on their own line instead of using the standard `else if`?